source/module_psi/psi_io.h

```diff
@@ -0,0 +1,258 @@
+/**
```

Comment:
I think we need to discuss whether to print out psi in this way

Reply:
> I think we need to discuss whether to print out psi in this way

@mohanchen I agree. The present format is not quite well-formatted that cannot be read by numpy.loadtxt, it would be more suitable to be printed to stdout but not too many people will care these details.